pkg/options/types.go

```diff
@@ -57,7 +57,7 @@ func (ms MetricSet) asSlice() []string {
 
 // IsEmpty returns true if the length of the MetricSet is zero.
 func (ms MetricSet) IsEmpty() bool {
```

Comment:
Let's remove this method as it's unused.

Reply:
I had a feeling that this was used. Let me double check...

Reply:
Removed!